src/AppMenuButtonGroup.cc

```
@@ -393,8 +398,11 @@ void AppMenuButtonGroup::resetButtons()
 
 void AppMenuButtonGroup::onMenuReadyForSearch()
```

Clang (14.0.6)

[warning] method 'onMenuReadyForSearch' can be made static

(readability-convert-member-functions-to-static)